### DISCUSSION TITLE
Add IsAccessible and ReadOnly flags to Get-DbaLastGoodCheckDb

### DISCUSSION THREAD

**Discussion by LeonardRutkowski** [dataplat/dbatools#9234] [Ideas]
Add IsAccessible and ReadOnly flags to Get-DbaLastGoodCheckDb. Would it be possible to add these two database items to the custom object? This would allow us to eliminate read only databases, when checking dbcc last good check.

**Comment by wsmelton** [comment]
I would say since this command is geared toward databases that it needs to output the full DB object and we add on the DBCC output as properties to that object.
